Rakefile

```diff
@@ -0,0 +1,20 @@
+require "rake"
+
+task :default => :build
+
+# rake build
+desc "Build the site"
+task :build do
+  shell_args = %w[bundle exec jekyll build]
+  sh *shell_args
```

Comment:
Probably worth just inlining this argument. Does Jekyll have any Rake helpers we can use?